### DISCUSSION TITLE
WMS 1.3.0: OGC requirements coverage in INSPIRE Validator

### DISCUSSION THREAD

**Discussion by ghost** [INSPIRE-MIF/helpdesk-validator#937] [Ideas]
Hi community,

I would like to test only OGC requirements for INSPIRE View Services. 

1. In which conformance class are OGC specific tests are covered? 
2. If possible, could you please reference executable tests which cover OGC specific requirements in the [ATS](https://github.com/inspire-eu-validation/view-service/tree/review-ats-tg_3.11/iso-19128)? 

Thanks in advance!

**Comment by fabiovin** [comment]
Dear @DeordD,

thank you for your question.

We’ll get back to you ASAP.

**Comment by fabiovin** [comment]
Dear @DeordD,

for INSPIRE View Services there are no dedicated conformance classes to test "only" the OGC requirements.

As indicated in the "[TG Requirement coverage](https://github.com/inspire-eu-validation/view-service/tree/review-ats-tg_3.11/iso-19128#tg-requirement-coverage)" section of the ATS, the requirements (1, 2 and 3) related to the use of ISO 19128 standard are not covered by specific tests but are "intrinsically" verified by the other tests.

You could use the [OGC Validator](https://cite.opengeospatial.org/teamengine/) to test only ISO/OGC requirements.

**Comment by ghost** [comment]
Dear @fabiovin,

thank you for your feedback. 

In INSPIRE Validator there there is external dependency to OGC Team Engine. OGC Conformance Class for Direct WFS 2.0 von OGC ("WFS 2.0 (OGC 09-025r2/ISO 19142) Conformance Test Suite")

![image](https://user-images.githubusercontent.com/38787131/133077491-f769f989-ab45-4e15-9c18-ce3f53c7ea13.png)

If tests are possible in OGC Team Engine, why does INSPIRE Validator doesn't use this Conformance Class for INSPIRE View services?

P.S.
Some discussion on this topic has already happened here https://github.com/opengeospatial/ets-wms13/issues/90

**Comment by ghost** [comment]
Hi,

Could someone provide some answer on the question below?

> If tests are possible in OGC Team Engine, why does INSPIRE Validator doesn't use this Conformance Class for INSPIRE View services?

Thanks!

**Comment by fabiovinci** [comment]
We considered this issue as a possible enhancement of the validator since requires the redesign of the WMS Conformance Class in order to include the related OGC test.
